janitor/functions/sort_column_value_order.py

```diff
@@ -48,19 +51,13 @@ def sort_column_value_order(
     :param columns: This is a list of additional columns that we can sort by
     :raises ValueError: raises error if chosen Column Name is not in
         Dataframe, or if column_value_order dictionary is empty.
-    :return: This function returns a Pandas DataFrame
+    :return: A sorted pandas DataFrame.
     """
+    check_column(df, column, present=True)
     if len(column_value_order) > 0:
```

Comment:
The way to implement guard clause for this particular if-else check will be something like this:

```python
check("column_value_order", column_value_order, [dict])  # I would probably just throw this in just in case
if not column_value_order:  # once we ensure this var is a dict, a "not var" is the pythonic way to check it is empty
    raise ValueError("column_value_order dictionary cannot be empty")
# then the remainder of the logic here
```

Reply:
Great catch, thanks @thatlittleboy!